backend/corpora/dataset_submissions/app.py

```diff
@@ -12,15 +12,14 @@
 from backend.corpora.common.utils.exceptions import CorporaException
 
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
```

Comment:
Removed hardcoded log level so that we can change it via the `LOGLEVEL` environment variable.

Reply:
Oops, sorry, my mistake... I misread an article about this. In hindsight, no, we'd have to read such an environment variable and then set the log level using `logging.setLevel`. Would be a handy feature to have!